### PR TITLE
Array support for IsEmpty rule

### PR DESCRIPTION
**Pull request by camcima**

### CODE REVIEW

File: src/Rule/NotEmpty.php [hunk 1]
```diff
@@ -105,7 +105,10 @@ public function validate($value)
      */
     protected function isEmpty($value)
     {
-        return (is_string($value) && strlen($value) === 0) || $value === null;
+        return
+            (is_string($value) && strlen($value) === 0) ||
+            $value === null ||
+            (is_array($value) && count($value) === 0);
```
Comment (RickvdStaaij) [inline src/Rule/NotEmpty.php]: I think it's good to add empty arrays here, but could you make this separate if statements? Thank you :)

**Comment by castarco** [comment]
Hi, what's the current state of this PR? The github notice-box tells that there aren't conflicts right now ^_^. If it's needed I can branch the @camcima 's work and split the return statement with an `if` statement to make it mergeable. I would love to have this feature merged in the Particle Validator.

**Comment by berry-langerak** [comment]
Oh my. Lost track of this, sorry. Looks good to go now, so merging.

**Comment by berry-langerak** [comment]
You can now use this in version 2.0.1.

**Pull request merged**